operator-framework/src/test/java/io/javaoperatorsdk/operator/sample/multiplesecondaryeventsource/MultipleSecondaryEventSourceReconciler.java

```diff
@@ -75,7 +77,7 @@ public Map<String, EventSource> prepareEventSources(
           return Set.of(new ResourceID(name.toString(), s.getMetadata().getNamespace()));
         }).build();
     InformerEventSource<ConfigMap, MultipleSecondaryEventSourceCustomResource> configMapEventSource =
-        new InformerEventSource<ConfigMap, MultipleSecondaryEventSourceCustomResource>(config,
+        new InformerEventSource<>(config,
             context);
```

Comment:
This should be put on the same line as the previous one, imo.

Reply:
fixed.